iFixit/react-commerce
# PPRO: Replace "Matomo" with "Piwik" In Most Instances

## Overview

Now that we are moving away from Matomo to Piwik, but many of our Matomo functions work for Piwik too, we are just going to change the function/variable/file names so that they are now "Piwik". Pretty straighforward.

We want to keep track of users preferred Store for their session. We can track that per user sessions by adding a custom dimension, in this case its preferredStore, to Piwik. This pull introduces the code to track that dimension in our code for each users session.

## QA

Make sure that the Piwik analytics still work.

Make sure that users session has the `preferredStore` as a dimension in the analytics.


Connects: #2016
